src/styles/index.ts

```diff
@@ -567,7 +567,6 @@ const styles = (theme: ThemeColors) =>
         },
 
         buttonContainer: {
```

Comment:
At this point I wonder, what's the point of the buttonContainer? Is it just for accessibility or something? Because the buttons themselves have the border radius applied... curious what you think.

Reply:
This one is being used here:

https://github.com/Expensify/App/blob/cba6916458c91e26ab8fb795956df0aa9508d6ab/src/components/Button/index.tsx#L327-L333

I see that here we set the disabled cursor if the button is disabled. If we remove the border radius, the `disabled` cursor would be displayed even if we move our pointer outside the rounded button, but inside the rectangle (the red areas):

<img width="211" alt="Screenshot 2024-05-14 at 17 23 25" src="https://github.com/Expensify/App/assets/104823336/4824895e-38db-4388-9927-e9894b737e1c">

Reply:
Cool, probably not worth touching this, was mostly just curious. Glad we got rid of the 1px padding though.